internal/resources/cluster/manifest/manifest.go

```diff
@@ -31,7 +31,7 @@ func Create(
 		return errors.WithMessage(err, "failure to fetch attach manifests")
 	}
 
-	toBeCleaned, err := objectsToBeCleaned(k8sclient, manifests, forceClean)
+	toBeCleaned, err := objectsToBeCleaned(*k8sclient, manifests, forceClean)
```

Comment:
objectsToBeCleaned(*k8sclient, manifests, forceClean)

*k8sclient, looks odd and should be avoided, so can you change the function definition for objectsToBeCleaned and 
createObjects